### PR TITLE
`treehouses help message` english (fixes #2049)

### LINKED ISSUE #2049: `treehouses message`  help description style overflow
The `treehouses message` help description overflows the standard 80 character column coding style. 
Following good coding style improves readability with small terminals😃.

* https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/modules/message.sh#L507
* https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/modules/message.sh#L509
* https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/modules/message.sh#L544-L545

### PR DESCRIPTION
**Pull request by rjpadilla**

### CODE REVIEW

File: modules/message.sh [hunk 1]
```diff
@@ -504,11 +504,21 @@ function message {
 
 function message_help {
   echo
-  echo "Usage: $BASENAME message <chats> <apitoken>|<oauth key> <redirect URL> | <authorize> <code> <oauth secret>| send <group> <message> | show|read|mark <group>"
+  echo "Usage: $BASENAME message <chats> 
+                          [<apitoken>|<oauth key> <redirect URL> | 
+                           <authorize> <code> <oauth secret> | 
+                           <send> <group> <message> | 
+                           <show|read|mark> <group>]"
```
Comment (dogi) [inline modules/message.sh:511]: @rjpadilla better to use in each line an own `echo`

**Review by JLKwong — CHANGES_REQUESTED** [review]
This line is wrapping off-screen for me when I do split screen in bash.
>   echo "  $BASENAME message gitter send treehouses/Lobby \"Hi, you are very awesome\""

**Comment by rjpadilla** [comment]
> 
> 
> This line is wrapping off-screen for me when I do split screen in bash.
> 
> > echo "  $BASENAME message gitter send treehouses/Lobby "Hi, you are very awesome""

So I tested this in git bash and it works fine. Maybe its a font size or monitor issue?

**Review by JLKwong — APPROVED** [review]
My font size is too big. D:

**Review by dogi — APPROVED** [review]

**Pull request merged**